docs/image_previews/readme.md

```diff
@@ -0,0 +1,58 @@
+# Image Thumbnails in File Previews
```

Comment:
This file should be all caps `README.md` just to be consistent :+1:

Reply:
Done.
Mostly, I have just re-arranged what was already there.  But thanks and welcome! :wink: